### PR TITLE
PXY-222 Add inject/remove header interceptor demo

### PR DESCRIPTION
**Pull request by laxuankhanh1902**

### CODE REVIEW

File: inject-remove-header/Readme.md [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+    }'
+```
+
+
+### Step 6: Produce data to the topic
+
+Let's produce a simple record to the `inject_header_topic` topic.
+
+```bash
+echo 'inject_header' | docker-compose exec kafka-client \
```
Comment (framiere) [inline inject-remove-header/Readme.md:132]: `docker compose exec -T`
it needs to open a tty

File: inject-remove-header/Readme.md [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+    }'
+```
+
+
+### Step 10: Produce data to the topic
+
+Let's produce a simple record to the `remove_header_key_pattern_topic` topic.
+
+```bash
+echo 'k0:v0,k1:v1\tkey_pattern' | docker-compose exec kafka-client \
```
Comment (framiere) [inline inject-remove-header/Readme.md:207]: same: `-T` for `docker compose`

File: inject-remove-header/Readme.md [hunk 3]
```diff
@@ -0,0 +1,491 @@
...
+    }'
+```
+
+
+### Step 14: Produce data to the topic
+
+Let's produce a simple record to the `remove_header_value_pattern_topic` topic.
+
+```bash
+echo 'k0:value0,k1:someValue\tvalue_pattern' | docker-compose exec kafka-client \
```
Comment (framiere) [inline inject-remove-header/Readme.md:284]: same: `-T` for `docker compose`

File: inject-remove-header/Readme.md [hunk 4]
```diff
@@ -0,0 +1,491 @@
...
+    }'
+```
+
+
+### Step 18: Produce data to the topic
+
+Let's produce a simple record to the `remove_header_key_value_pattern_topic` topic.
+
+```bash
+echo 'k0:v0,k1:v1\tkey_value_pattern' | docker-compose exec kafka-client \
```
Comment (framiere) [inline inject-remove-header/Readme.md:362]: same: `-T` for `docker compose`

**Review by framiere — CHANGES_REQUESTED** [review]
LGTM, just missing `-T` in `docker compose` when using TTY

File: inject-remove-header/Readme.md (hunk 1 above)
Comment (laxuankhanh1902) [reply inject-remove-header/Readme.md]: Thank you, I updated

File: inject-remove-header/Readme.md (hunk 3 above)
Comment (laxuankhanh1902) [reply inject-remove-header/Readme.md]: Thank you, I updated

File: inject-remove-header/Readme.md (hunk 4 above)
Comment (laxuankhanh1902) [reply inject-remove-header/Readme.md]: Thank you, I updated

**Review by framiere — APPROVED** [review]
LGTM

**Pull request merged**